articles/vmware/vmw-how-vls-install-usage-meter-4.md

```diff
@@ -0,0 +1,124 @@
+
+## Prerequisites
+
+To install and perform initial configuration of the Usage Meter, you need the following:
+
+- Usage Meter software, available [here](https://cas.cor00005.ukcloud.com/Docs/UKCloud_VLS/UsageMeter4x.zip?AWSAccessKeyId=438-1048-5-aefff7-1&Expires=1674593692&Signature=lQPfSuNylwxGS8bd9E%2BIG8L4BcA%3D)
+
+- A Static IP address, DNS server IP address and DNS domain name
+
+- A vCenter network that has access to vCenter management IP and other products to be metered
+
+- Internet access via port 443 to the URL **ums.cloud.vmware.com** for the usage meter IP
+
+- If using vCenter version 6.5, please see note regarding a potential deployment issue and workaround [here](https://kb.vmware.com/s/article/85154)
+
+- Three passwords created and recorded for the usage meter's `root`, `usagemeter` and `umauditor` accounts
+
+### Deploying and setting up the Usage Meter appliance
+
+To deploy and set up the Usage Meter:
+
+1. Download and unzip the Usage Meter OVA file from the link provided in the [Prerequisites](#prerequisites) section.
+
+2. In vCenter, navigate to the folder where you want to deploy the Usage Meter.
+
+3. From the **Actions** menu, select **Deploy OVF Template**.
+
+4. In the *Deploy OVF Template* dialog box, select **Local file** and browse for the Usage Meter OVA file, then click **Next**.
+
+5. Enter a **Name** for the VM and ensure the correct folder is selected (if not, select it now), then click **Next**.
+
+6. Select a cluster or ESXi server to host the VM, then click **Next**.
+
+7. Review the OVA file details, then click **Next**.
+
+8. Review the licence agreement and click **Accept**, then click **Next**.
+
+9. Select the desired data store on which to deploy the VM and, from the **Select virtual disk format** list, select **Thin Provision**, then click **Next**.
+
+10. Select the network you want the Usage Meter to reside on and, from the **IP Allocation** list, select **Static - Manual**.
+
+11. On the *Customize template* page, set the passwords to use for the Usage Meter `root` and `usagemeter` and `umauditor` accounts.
+
+12. Depending on the vCenter client used, expand the *Networking Properties* section and enter the following information (where possible), then click **Next**.
+
+    - **Hostname**: Host name of the usage meter appliance
+    
+    - **Host Network Default Gateway**: Gateway IP for the network where the Usage Meter is located
+
+    - **Domain name**: The domain name of this VM
+
+    - **DNS search path**: Use the AD Domain name
+
+    - **Domain Name Servers**: The domain name servers IP Addresses for this VM
+    
+    - **Network 1 IP Address**: Enter the IP address to assign to the Usage Meter
+
+    - **Network 1 Netmask. Netmask in CIDR notation.**: The network mask for this interface. Netmask in CIDR notation (e.g. input 24 for 255.255.255.0, 28 for 255.255.255.240)
```

Comment:
Could this just be:
- **Network 1 Netmask**: The network mask for this interface in CIDR notation (for example, enter 24 for 255.255.255.0 or 28 for 255.255.255.240)

Reply:
yep, fine with me, i lifted straight from VMware docs page.

Reply:
Changed it